Bottom: 849ae72d6f3457d6b4f76f54a366ca880bf50cf6
Top:    a15fbbe5bade08e16e734978a7291de5da535e96
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:24:34 +0000

Disable process limits


---

diff --git a/src/polygon.c b/src/polygon.c
index 420772b..fb40faf 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -132,6 +132,7 @@ polygon_init (void)
 
   bw_rotate_circle_seg[0] =  cos_ang;  bw_rotate_circle_seg[1] =  sin_ang;
   bw_rotate_circle_seg[2] = -sin_ang;  bw_rotate_circle_seg[3] =  cos_ang;
+return;
 
   /* DEBUG - AVOID PCB running the system out of memory! */
   getrlimit (RLIMIT_AS, &limit);
